### PR TITLE
Fix large job configs

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

File: jobs/ci-kubernetes-e2e-gke-large-deploy.env [hunk 1]
```diff
@@ -17,6 +17,5 @@ GKE_CREATE_FLAGS=--max-nodes-per-pool=1000
 CLOUDSDK_CONTAINER_USE_CLIENT_CERTIFICATE=True
 CLOUDSDK_API_ENDPOINT_OVERRIDES_CONTAINER=https://staging-container.sandbox.googleapis.com/
 KUBE_GKE_IMAGE_TYPE=gci
-E2E_DOWN=false
```
Comment (wojtek-t) [inline jobs/ci-kubernetes-e2e-gke-large-deploy.env]: @krzyzacy - it seems those no longer work. PTAL if this PR makes sense.

Comment (krzyzacy) [inline jobs/ci-kubernetes-e2e-gke-large-deploy.env]: oh sorry, it's the same issue as kubemark, good catch.

File: jobs/config.json [hunk 2]
```diff
@@ -1693,15 +1696,18 @@
   "scenario": "kubernetes_e2e",
   "args": [
     "--env-file=platforms/gke.env",
-    "--env-file=jobs/ci-kubernetes-e2e-gke-large-deploy.env"
+    "--env-file=jobs/ci-kubernetes-e2e-gke-large-deploy.env",
+    "--down=false"
```
Comment (krzyzacy) [inline jobs/config.json]: oh, add --cluster=gke-large-deploy here and remove E2E_NAME env, as I forgot in kubemark. Same for rest of jobs.

**Comment by krzyzacy** [comment]
I will update unit-test to eliminate E2E_NAME|UP|TEST|DOWN in the env, or else it's hard to cover everything.

**Comment by krzyzacy** [comment]
close via #1836

**Pull request closed without merging**